frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -221,23 +225,122 @@ export class HtmlDocument extends React.Component {
     if (prevState.content.content_id !== state.content.content_id) {
       await this.loadContent()
       globalThis.tinymce.remove('#wysiwygNewVersion')
-      globalThis.wysiwyg('#wysiwygNewVersion', state.loggedUser.lang, this.handleChangeText)
+      globalThis.wysiwyg('#wysiwygNewVersion', state.loggedUser.lang, this.handleChangeText, this.handleTinyMceInput, this.handleTinyMceKeyDown)
     }
 
     if (state.mode === APP_FEATURE_MODE.EDIT && prevState.mode !== APP_FEATURE_MODE.EDIT) {
+      globalThis.tinymce.remove('#wysiwygTimelineComment')
       globalThis.tinymce.remove('#wysiwygNewVersion')
-      globalThis.wysiwyg('#wysiwygNewVersion', state.loggedUser.lang, this.handleChangeText)
+      globalThis.wysiwyg('#wysiwygNewVersion', state.loggedUser.lang, this.handleChangeText, this.handleTinyMceInput, this.handleTinyMceKeyDown)
     }
 
-    if (!prevState.timelineWysiwyg && state.timelineWysiwyg) globalThis.wysiwyg('#wysiwygTimelineComment', state.loggedUser.lang, this.handleChangeNewComment)
-    else if (prevState.timelineWysiwyg && !state.timelineWysiwyg) globalThis.tinymce.remove('#wysiwygTimelineComment')
+    if (!prevState.timelineWysiwyg && state.timelineWysiwyg) {
+      globalThis.tinymce.remove('#wysiwygNewVersion')
+    } else if (prevState.timelineWysiwyg && !state.timelineWysiwyg) globalThis.tinymce.remove('#wysiwygTimelineComment')
 
     // INFO - CH - 2019-05-06 - bellow is to properly init wysiwyg editor when reopening the same content
     if (!prevState.isVisible && state.isVisible) {
       initWysiwyg(state, state.loggedUser.lang, this.handleChangeNewComment, this.handleChangeText)
     }
   }
 
+  handleInitTimelineCommentWysiwyg = (handleTinyMceInput, handleTinyMceKeyDown) => {
+    globalThis.wysiwyg('#wysiwygTimelineComment', this.state.loggedUser.lang, this.handleChangeNewComment, handleTinyMceInput, handleTinyMceKeyDown)
+  }
+
+  handleTinyMceInput = (e, position) => {
+    if (!e.data) return
+
+    switch (e.data) {
+      case '@': {
+        const rawHtml = (
+          '<span id="autocomplete">' +
```

Comment:
you can use backticks (string interpolation) do declare multiline jsx instead of concatinating single quote strings